components/31-molecules/modal/_modal.scss

```diff
@@ -5,6 +5,7 @@
   right: 0;
   bottom: 0;
   left: 0;
+  max-height: 100vh;
```

Comment:
Have you checked on mobile?
This gives issues with the URL toolbar.

Reply:
Well, I've tested it on multiple OS / browsers:

* Android (Chrome)
* iOS (Chrome / Safari)
* OSX (Chrome / Safari / Firefox)
* Windows (IE 11)